readme.md

```diff
@@ -98,6 +98,15 @@ Type `RegExp`
 Ignore lines matching the given regex.
 
 
+## Toggle loggers
```

Comment:
@lukechilds Suggestions for a better title?

Reply:
I'm awful at naming things but this seems.o to me.